### PR TITLE
Added warnings for unavailable sections (See #361)

### LINKED ISSUE #361: You can add unavailable lecture sections
On VIAPlanner you're able to select and add lecture and tutorial sections that are unavailable to your schedule. For example, CSC236H5F's lecture 0103 and tutorial 108 are both unavailable on ttb, but you can add them to your schedule on VIAPlanner.

### PR DESCRIPTION
**Pull request by ggggg**
## feat/[name]

Fixes: #361

### CODE REVIEW

File: src/components/Popup/CourseSectionPicker.vue [hunk 1]
```diff
@@ -66,7 +66,7 @@
                           <v-row class="center-vertical">
                             <v-col>
                               <v-list-item-title>
-                                {{ meetingSection.sectionCode }}
+                                <warning v-if='getWarningSections.some(x=>x.code === course.courseCode && x.sectionCode === meetingSection.sectionCode)'/>{{ meetingSection.sectionCode }}
```
Comment (logonoff) [inline src/components/Popup/CourseSectionPicker.vue:69]: thats a long line

**Review by logonoff — APPROVED** [review]

**Comment by huonggiangbui** [comment]
For future PR, please include attachment (e.g. demo of UI) so it's easier to visualize!

File: src/components/Popup/CourseSectionPicker.vue (hunk 1 above)
Comment (huonggiangbui) [reply src/components/Popup/CourseSectionPicker.vue:69]: Agree

File: src/components/SidePanel/Warning.vue
Comment (huonggiangbui) [inline src/components/SidePanel/Warning.vue:1]: should add EOF line (new line at end of file)

**Comment by ggggg** [comment]
> For future PR, please include attachment (e.g. demo of UI) so it's easier to visualize!

image

File: src/store/index.js
Comment (huonggiangbui) [inline src/store/index.js:1]: I think hash map is a bit faster for this, instead of using set of string

**Review by huonggiangbui — APPROVED** [review]
Few things can be improved, otherwise LGTM

File: src/components/SidePanel/Warning.vue
Comment (ggggg) [reply src/components/SidePanel/Warning.vue:1]: Should probably add an issue for eslint linting then

File: src/store/index.js
Comment (ggggg) [reply src/store/index.js:1]: Can you describe your implementation

File: src/components/SidePanel/Warning.vue
Comment (huonggiangbui) [reply src/components/SidePanel/Warning.vue:1]: Yup!

File: src/store/index.js
Comment (huonggiangbui) [reply src/store/index.js:1]: Will get to it tonight!

**Pull request merged**